types/rc-slider/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for rc-slider 8.2
+// Type definitions for rc-slider 9.0
```

Comment:
Is this PR reflecting a breaking change in the `rc-slider` package? That package only seems to be at version 8.6. 
If it really is a breaking change, then there's [some extra steps](https://github.com/DefinitelyTyped/DefinitelyTyped#i-want-to-update-a-package-to-a-new-major-version) you'll need to take.

Reply:
No, this number should follow the version number of the rc-slider package. Thank you for being careful about this, though!